lib/async/container/supervisor/server.rb

```diff
@@ -80,6 +86,8 @@ def run(parent: Task.current)
 						@endpoint.accept do |peer|
 							connection = Connection.new(peer, 1)
 							connection.run(self)
+						rescue => error
+							Console.error(self, "Error while accepting connection!", exception: error)
```

Comment:
I think this one is the most important, or else any error will silently kill the Fiber here: https://github.com/socketry/io-endpoint/blob/main/lib/io/endpoint/wrapper.rb#L169